SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -15,7 +15,9 @@ void USpatialWorkerConnection::SetConnection(Worker_Connection* WorkerConnection
 
 	CacheWorkerAttributes();
 
-	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();    
+	const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>();
+	bool bCanWake = SpatialGDKSettings->bWorkerFlushAfterRPC;
+	ThreadWaitCondition.Emplace(bCanWake, OpsUpdateInterval);
```

Comment:
Shouldn't we only set this up if `!SpatialGDKSettings->bRunSpatialWorkerConnectionOnGameThread` ?

Reply:
Yup!

Reply:
Yup!